operator-common/src/main/java/io/strimzi/operator/common/ConfigUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017-2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.common;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Utility methods for Configration parsing and handling
+ */
+public class ConfigUtils {
+    /**
+     * Parse Labels from String into Map. The expected format of the String with labels is `key1=value1,key2=value2`
+     *
+     * @param stringLabels  String with labels
+     * @return  Map with parsed labels
+     * @throws InvalidConfigurationException
+     */
+    public static Map<String, String> parseLabels(String stringLabels) throws InvalidConfigurationException {
```

Comment:
Now I look at this method, and alluding to my earlier comment about `Labels` class, maybe we should put this there, rather than introducing a new Utils class for it?

Reply:
I'm not completely sure it belongs there. It is used only for parsing of config when starting the operators. `Labels` are used all over the place. But if you want I can move it there.

If we put it to `Labels` it should probably return `Labels` and not map, right? And we should probably also remove the InvalidConfigException from there.

Reply:
We pass `Labels` around, but conceptually it's just a `Map<String, String>` and that's what we always have to convert it into when we use the Fabric8 API. What `Labels` give us it an immutable representation of that Map, which prevents people doing nasty things like mutating a `Labels` they were passed. I suppose its immutability is useful enough that it's worth keeping. A `Labels.parseLabels()` should return a Labels instance. 

I don't understand your point about `InvalidConfigException`.

Reply:
OIC. Yes that's right.